lib/ansible/modules/cloud/amazon/rds.py

```diff
@@ -123,7 +123,7 @@
     default: 3306 for mysql, 1521 for Oracle, 1433 for SQL Server, 5432 for PostgreSQL.
   upgrade:
     description:
-      - Indicates that minor version upgrades should be applied automatically. Used only when command=create or command=replicate.
+      - Indicates that minor version upgrades should be applied automatically. Used only when command=create, command=modify, or command=replicate.
```

Comment:
Is it valid for restore as well?

Reply:
I don't know and I'm not sure when I'll be able to test again. I've moved on to other projects since submitting this PR

Reply:
Understood - thanks @ktstevenson.